src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -267,7 +267,7 @@ internal struct PointerEventDispatchResult
 		/// </summary>
 		internal HitTestability GetHitTestVisibility()
 		{
-#if __WASM__ || __SKIA__ || __MACOS__
+#if UNO_HAS_ENHANCED_HIT_TEST_PROPERTY
```

Comment:
This doesn't include macOS I think

Reply:
Indeed ... which means we had a bug on macOS. Fixed by removing the UNO_HAS_ENHANCED_HIT_TEST_PROPERTY and use only UNO_HAS_MANAGED_POINTERS which is now the same (and has to be).